client/starwhale/core/model/cli.py

```diff
@@ -140,6 +140,11 @@ def _extract(model: str, force: bool, target_dir: str) -> None:
 )
 @click.option("--step", default="", help="Evaluation run step")
 @click.option("--task-index", default=0, help="Index of tasks in the current step")
+@click.option(
+    "--override-task-num",
+    envvar=SWEnv.total_task_num,
```

Comment:
task_num env will be effective on all steps under the standalone mode.

Reply:
done